Modules/Map/WeaponMasterSkills.lua

```diff
@@ -0,0 +1,23 @@
+local WeaponMasterSkills = {}
```

Comment:
Please don't do this 🙈 

Checkout other "modules" which are using the `QuestieLoader` global to create and import modules.

Reply:
This should now be fixed.